mythril/laser/ethereum/instructions.py

```diff
@@ -1609,7 +1571,31 @@ def delegatecall_post(self, global_state: GlobalState) -> List[GlobalState]:
     def staticcall_(self, global_state: GlobalState) -> List[GlobalState]:
         # TODO: implement me
         instr = global_state.get_current_instruction()
+        try:
+            callee_address, callee_account, call_data, value, call_data_type, gas, memory_out_offset, memory_out_size = get_call_parameters(
+                global_state, self.dynamic_loader
+            )
+        except ValueError as e:
+            log.debug(
+                "Could not determine required parameters for call, putting fresh symbol on the stack. \n{}".format(
+                    e
+                )
+            )
+            global_state.mstate.stack.append(
+                global_state.new_bitvec("retval_" + str(instr["address"]), 256)
+            )
+            return [global_state]
+
         global_state.mstate.stack.append(
             global_state.new_bitvec("retval_" + str(instr["address"]), 256)
         )
+        if 0 < int(callee_address, 16) < 5:
```

Comment:
can this line be somehow be merged into native_call, or other way is can you use a constant for the 5, as in the future we must increase the native contracts count, so we might need to change this number in multiple places, so localizing it might be better.